components/entitlement/org.wso2.carbon.identity.entitlement.ui/src/main/resources/web/entitlement/add-subscriber.jsp

```diff
@@ -305,12 +306,14 @@
                     </td>
                     <td>
                         <% if(dto.getValue() != null) {%>
-                            <input type="<%=inputType%>" name="<%=dto.getId()%>" id="<%=dto.getId()%>"
-                                                                        value="<%=dto.getValue()%>" <% if("subscriberId".equals(dto.getId())){ %> readonly='readonly' <% } %> />
+                            <input type="<%=Encode.forHtmlAttribute(inputType)%>" name="<%=Encode.forHtmlAttribute(dto.getId())%>"
```

Comment:
Here inputType has hardcoded values which does not include special characters and trusted. Thus, it's not necessary to encode this variable